src/main/java/htsjdk/samtools/cram/ref/ReferenceContext.java

```diff
@@ -0,0 +1,141 @@
+package htsjdk.samtools.cram.ref;
+
+import htsjdk.samtools.cram.CRAMException;
+import htsjdk.samtools.cram.structure.Slice;
+
+/**
+ * Represents the current state of reference sequence processing.
+ *
+ * Are we handling MULTI_REFERENCE records (-2, from the CRAM spec)?
+ *
+ * Are we handling UNMAPPED_UNPLACED records (-1, from the CRAM spec)?
+ *
+ * Or are we handing a known SINGLE_REFERENCE sequence (0 or higher, from the CRAM spec))?
+ *
+ */
+public class ReferenceContext implements Comparable<ReferenceContext> {
+    // TODO: merge with Slice constants
```

Comment:
Is there any reason we can't do this now (see my comments below) ? It seems like the ultimate authority for these values should be `ReferenceContext` ?

Reply:
Nope.  Done now.  I'm guessing something in a previous iteration made this easier.